tests/utils/eval_client/test_lambda_eval_client.py

```diff
@@ -0,0 +1,37 @@
+# Copyright 2022 MosaicML Composer authors
+# SPDX-License-Identifier: Apache-2.0
+import pytest
+
+from composer.utils import LambdaEvalClient
+
+
+@pytest.mark.daily
```

Comment:
```suggestion
```
Let's leave as remote only for now. currently, remote implies daily, but this may change in future. I want to separate remote vs. daily that are just slow. Sorry this is confusing right now...

Reply:
ah ok